src/Microsoft.TestPlatform.Common/Microsoft.TestPlatform.Common.csproj

```diff
@@ -34,6 +34,9 @@
     <ProjectReference Include="..\Microsoft.TestPlatform.CoreUtilities\Microsoft.TestPlatform.CoreUtilities.csproj">
       <FromP2P>true</FromP2P>
     </ProjectReference>
+    <PackageReference Include="Newtonsoft.Json">
+      <Version>$(JsonNetVersion)</Version>
```

Comment:
Did you analyze the impact of this? Why is it safe?

Reply:
@nohwnd the only other project that depends on `Newtonsoft.Json` is `Microsoft.TestPlatform.CommunicationUtilities`, so I checked that both `Microsoft.TestPlatform.CommunicationUtilities` and `Microsoft.VisualStudio.TestPlatform.Common` are packaged together in the following packages (so the dependency requirements were already in place in each `.nuspec`):

- `Microsoft.TestPlatform.TranslationLayer`
- `Microsoft.TestPlatform.Internal.Uwp`
- `Microsoft.TestPlatform`
- `Microsoft.TestPlatform.Portable`

Because of this, I reconsidered the previous position where I didn't want to take a dependency on `Newtonsoft.Json`.

Other than that, I did manual validation for the metrics format and here's a sample of how it looks:

```
{
   "TestSettingsProviders":[
      "WebTestRunConfiguration",
      "LegacySettings",
      "MSTest"
   ],
   "TestHosts":[
      "HostProvider://DefaultTestHost",
      "HostProvider://DotnetTestHost"
   ]
}
```

Do you have any particular checks in mind ?

Reply:
This is not problematic for projects that already depend on the library you are adding, as long as they both depend on the same version. 

This is problematic for projects that don't depend on it. Did you check if all the projects that depend on common already depend on newtonsoft.json and ship it?

Reply:
Moved to `SimpleJSON`, resolving the conversation.